gdsc-web-backend/Controllers/TeamController.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System.Collections.Generic;
+using gdsc_web_backend.Models;
+using Microsoft.AspNetCore.Mvc;
+
+namespace gdsc_web_backend.Controllers
+{
+    [ApiController]
+    [Route("api/[controller]")]
+    
+    public class TeamController : ControllerBase
```

Comment:
Please rename Team Controller to TeamsController, cause a controller manages more than a team